partners.yml

```diff
@@ -523,6 +523,19 @@ components:
           $ref: "#/components/schemas/bankDetails"
         employer:
           $ref: "#/components/schemas/employer"
+        comment:
+          type: array
+          items:
+            type: object
+            properties:
+              comment:
+                type: string
+                description: Comments that underwriter or the system can add for describing specific events
```

Comment:
`Comments that underwriters or the system can use as notes for the applicant or application`